integration_tests/e2e/personal/edit/editDomesticStatus.cy.ts

```diff
@@ -18,7 +18,7 @@ context('Edit domestic status', () => {
     bookingId,
     testSetup: () => {
       cy.task('reset')
-      cy.setupUserAuth({ roles: [Role.PrisonUser, 'DPS_APPLICATION_DEVELOPER'] })
+      cy.setupUserAuth({ roles: [Role.PrisonUser] })
```

Comment:
As per https://dsdmoj.atlassian.net/browse/CDPS-1460, domestic status is a non-sensitive edit field